app/javascript/pages/form/components/listingApplyFormWrapper.scss

```diff
@@ -18,4 +18,65 @@
     }
     text-transform: uppercase;
   }
+
+  .application-form-nav {
+    overflow: auto;
+    padding: var(--seeds-s4) var(--seeds-s2) var(--seeds-s6);
+    @media (max-width: theme("screens.md")) {
+      background-color: var(--seeds-color-blue-100);
+      padding: var(--seeds-s4) 0 var(--seeds-s6);
+    }
+    .progress-nav {
+      width: 100%;
+    }
+
+    .progress-nav__item-container {
+      padding-top: var(--seeds-s5);
+      padding-left: var(--seeds-s2_5);
+    }
+
+    .progress-nav__dot-item {
+      text-align: center;
+      top: var(--seeds-s0_5);
+
+      &:before {
```

Comment:
Curious if there is a preference here using scss parent selectors (&:before) versus traditional css nesting (i.e. .progress-nav__dot-item:before)